src/components/UI/Text/Textarea.vue

```diff
@@ -220,5 +233,32 @@ export default {
     },
   },
   emits: ["input", "keypress", "keydown", "update:value", "start-icon-selected"],
+  // quilljs editor
```

Comment:
This comment is above the `mounted`.
Currently, in `mounted`, the only thing happening is setting up of quill editor.
But other things can also happen, not related to quill editor.
So keeping a comment `quilljs editor` above `mounted` is not correct as `mounted` is not something that we have defined. It's already there. We're just doing a bunch of stuff when component is mounted.

You should move this comment inside, just above the code where you're dealing with Quill editor. And write the comment in a little more detail.